src/server/pkg/worker/master.go

```diff
@@ -387,6 +399,7 @@ func (a *APIServer) runJob(ctx context.Context, jobInfo *pps.JobInfo, pool *grpc
 		}
 		// set the initial values
 		updateProgress(0)
+		protolion.Infof("Updated initial job state (%v)\n", jobInfo)
```

Comment:
We don't need a log line to tell us we update progress... we'll be able to see that by checking progress.